spdx-utils/src/main/kotlin/Extensions.kt

```diff
@@ -26,6 +26,20 @@ import java.util.EnumSet
  */
 operator fun <E : Enum<E>> EnumSet<E>.plus(other: EnumSet<E>): EnumSet<E> = EnumSet.copyOf(this).apply { addAll(other) }
 
+infix fun SpdxExpression.and(other: SpdxExpression) =
```

Comment:
Please add docs for these to keep everything in this file documented.

Reply:
Done, and moved some functions to `SpdxExpression`.